.github/workflows/IntegrationTesting.yml

```diff
@@ -0,0 +1,140 @@
+name: Integration Testing
+on: push
```

Comment:
I feel we can right now run on push only changes and then we can introduce on PRs as well. I think that is something we can wait on and collect some data points on what would be more useful. Ideally running on PR would be causing more stress on single aws account (in terms of ELB and other resources) because tests will run on every commit.